### PR TITLE
fix #8879

### LINKED ISSUE #8879: Meta, atmos entry room lacks a scrubbadub scrubber
(WEB REPORT BY: probe1 REMOTE: 158.69.55.2:7777)
http://i.imgur.com/Eql0sPS.png

3 atmos techs in a tub.  Plasma goes blub blub blub.  Without their glubs and stuck in the tubs, it needs a scrub scrub scrub.

### PR DESCRIPTION
**Pull request by Intigracy**
metashit

### CODE REVIEW

File: maps/metaclub.dmm [hunk 1]
```diff
@@ -4290,16 +4290,16 @@
...
+"bEF" = (/obj/machinery/camera{c_tag = "Atmospherics - Access"; network = list("SS13")},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,/turf/simulated/floor,/area/engineering/atmos)
+"bEG" = (/turf/simulated/floor{icon_state = "caution"; dir = 4},/area/engineering/atmos_control)
+"bEH" = (/obj/machinery/meter{frequency = 1443; id_tag = "wloop_atm_meter"; name = "Waste Loop"},/obj/machinery/atmospherics/pipe/manifold/scrubbers/visible{dir = 8},/turf/simulated/floor,/area/engineering/atmos)
+"bEI" = (/obj/structure/cable/yellow{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/atmospherics/pipe/manifold/supply/hidden{dir = 8},/turf/simulated/floor,/area/engineering/atmos)
+"bEJ" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 10},/turf/simulated/floor,/area/engineering/atmos_control)
+"bEK" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 4; scrub_N2O = 1; scrub_Toxins = 0},/turf/simulated/floor,/area/engineering/atmos_control)
```
Comment (PJB3005) [inline maps/metaclub.dmm]: What the...

`scrub_N2O = 1; scrub_Toxins = 0`

Comment (Intigracy) [inline maps/metaclub.dmm]: stupid list being full of so many instances jfc.

**Comment by ghost** [comment]
Won't close the issue

**Comment by Intigracy** [comment]
Commit messages work too you moron.

**Pull request merged**